### DISCUSSION TITLE
Hardware upgrade

### DISCUSSION THREAD

**Discussion by inferno1231** [awawa-dev/HyperHDR#513] [Q&A]
Hello,
So some time passed since I have checked anything regarding my ambient lights setup.. it works and it is awesome. I use Feintech splitter connected to second Denon AVR output and Rullz MS2109 grabber. 
The only thing missing in my setup is ability to use PS5 at full power including VRR etc., right now if I turn on any in-game graphics that is not locked to either 30 or 60 fps leds start flickering hard.

So I decided to check if anything new came to the market. 
Seems like first properly working HDMI 2.1 Splitter appeared - https://amzn.eu/d/6Poxm2r
It should support 4K 120 HZ on both outputs, with ability to scale second output to 1080p 120 HZ and it also supports VRR, CEC etc. 

Any tips on grabber that would match this splitter? Is Elgato HD60x the only option now to fully support all tech? Or is there something less pricey? Am I correct thinking that I need a grabber that will support at least passthrough of 1080p120 stream? I don't think there is a grabber that supports 4k120 yet?

**Comment by awawa-dev** [comment]
Hi
You can check out the Ezcap 360 https://ezcap.com/index.php/product/ezcap360gamedockextreme.html Since the last experience with the two Ezcap products has been rather disappointing for me, and since I already have an Elgato HD60X that causes absolutely no problems, I rather skip purchase and review of this product: too much money to risk. Also judging by the price evolution of the Ezcap 320/326 it probably won't be much cheaper than the Elgato but so what: Elgato offers top build quality (my Ezcap 320 USB-C port already malfunctions/intermittently fails to contact) and support. Even if there are some glitches in the firmware, they are fixed almost immediately by Elgato.

**Reply by inferno1231** [reply]
Great to hear that Elgato works flawlessly. So just to make sure, will the following installation work correctly? PS5 -> Denon AVR -> Ezcoo Splitter - OUT1 4k120hz to TV, OUT2 scaled to 1080p120hz to HD60x? 
By the way, can I capture 1080p120 stream using Pi4? Or it works on Windows only?

**Reply by awawa-dev** [reply]
Don't have any experience with 4k120: only my Denon x2700 supports it (well, it has a HDMI2.1 BUG and they won't fix it for me) and the rest is only HDMI2.0.

About 1080p120...you can capture such stream on Rpi 4:
![obraz](https://user-images.githubusercontent.com/69086569/219903175-8d296acd-5c8f-494d-bfbc-04153a2f61ef.png)
But it's not recommended to use such high resolution for the ambient lighting.

**Reply by inferno1231** [reply]
I heard that Denon provides free adapter to bugged machines? Check here: https://www.denon.com/en-gb/hdmiadapter

**Reply by awawa-dev** [reply]
I tried, it's not that easy unfortunately.

**Reply by d22b** [reply]
> I heard that Denon provides free adapter to bugged machines? Check here: https://www.denon.com/en-gb/hdmiadapter

Please keep also in mind, that various Denon Receiver doing some weird stuff when both hdmi outputs are connected to different screens running at the same time. I’ve build some „Denon picture preview“ with one Output was connected to projector and one to an ms2103 grabber plugged in an raspi. The result was that the maximum resolution was 1080p on both outputs instead one 4K for Beamer and 1080 for raspi.

I was not able to change it separately.

**Reply by awawa-dev** [reply]
Yes, the same happens to me. The only possible workaround I found is to connect ms2130/09 to the splitter (as splitter secondary device, it's the only device connected to HDMI splitter output), then splitter to the Denon (so Denon has two outputs: tv + splitter/ms2130). The splitter must have EDID cloned from the tv or set by using switches. Denon internal HDMI splitter is almost useless in such configuration.

**Reply by inferno1231** [reply]
Yes, that is why I am using Feintech splitter on the second output of Denon - I am getting 4K 60hz on both outputs. The only way to bypass splitter would be a capture card that supports 4k60/4k120 depending on the needs. That's why I am excited about hdmi 2.1 splitters.

Edit:
Oops, missed @awawa-dev reply where he tells same stuff.

**Reply by d22b** [reply]
No problem. As awawa already said, they will most likely appears soon or later. But even the forerunner blackmagic and hdfury doesn’t offer full hdmi 2.1 support at all + they are brutal expensive.

It’s a shame that Denon doesn’t implement it at all in their AVRs. I mentioned that since two years but didn’t get an statement from them.

**Comment by awawa-dev** [comment]
BTW 4k120 is HDMI 2.1 Sooner or later the HDMI 2.1 grabbers will appear but currently the price would kill most of the potential clients. At least such speculations I found in response to the criticism of one manufacturer why they don't produce them yet (> 500 euros)
